src/Application/Console/Definitions/AnalyseDefinition.php

```diff
@@ -17,9 +17,9 @@ public static function get(): InputDefinition
     {
         return new InputDefinition([
             new InputArgument(
-                'directory',
+                'directories',
                 InputArgument::OPTIONAL | InputArgument::IS_ARRAY,
-                'The directory to analyse'
+                'The directories to analyse'
```

Comment:
```suggestion
                'The directories or files to analyse'
```

As we can pass also file path. Maybe we should rename argument into `paths` instead `directories` ?

Reply:
Could be done in another PR BTW

Reply:
Yep I agree. I try to do that in current PR.